website/static/js/file-browser.js

```diff
@@ -0,0 +1,284 @@
+/**
+ * Builds full page project browser
+ */
+'use strict';
+
+var Treebeard = require('treebeard');   // Uses treebeard, installed as plugin
+var $ = require('jquery');  // jQuery
+var m = require('mithril'); // exposes mithril methods, useful for redraw etc.
+var ProjectOrganizer = require('js/project-organizer');
+var $osf = require('js/osfHelpers');
+
+/**
+ *  Options for fileBrowser
+ */
+var defaults = {
+    wrapper : '#fileBrowser',  // Default ID for wrapping empty div, all contents will be filled in.
+    tboptions : {},
+    fullWidth : true
+};
+
+/**
+ * Initialize File Browser. Prepeares an option object within FileBrowser
+ * @constructor
+ */
+var FileBrowser = {
+    controller : function (args) {
+        var self = this;
+        self.data = [];
+        self.isLoadedUrl = false;
+
+        // View States
+        self.showInfo = m.prop(false);
+
+        self.collections = [
+            { id:1, type : 'collection', label : 'All My Projects', path : 'users/me/nodes/', pathOptions : {  query : { 'filter[registration]' : 'false'} } },
```

Comment:
Will `pathOptions` ever be anything but query string key/value pairs? There may unnecessary nesting here.